sdk/compute/Azure.ResourceManager.Compute/src/Generated/CloudServiceData.cs

```diff
@@ -20,6 +19,7 @@ public partial class CloudServiceData : TrackedResourceData
         /// <param name="location"> The location. </param>
         public CloudServiceData(AzureLocation location) : base(location)
         {
+            Zones = new ChangeTrackingList<string>();
         }
```

Comment:
i see much of the properties in this is now in CloudServiceProperties instead. Is this a problem?